website/docs/gitopssets/_api.md

```diff
@@ -0,0 +1,1079 @@
+<h1>Source API reference</h1>
+<p>Packages:</p>
+<ul class="simple">
+<li>
+<a href="#templates.weave.works%2fv1alpha1">templates.weave.works/v1alpha1</a>
+</li>
+</ul>
+<h2 id="templates.weave.works/v1alpha1">templates.weave.works/v1alpha1</h2>
+<p>Package v1alpha1 contains API Schema definitions for the gitopssets v1alpha1 API group</p>
+Resource Types:
+<ul class="simple"></ul>
```

Comment:
Maybe we can populate this list by adding comments some of the types a bit more? Well.. I guess the only resourcetype is the gitopsset really though.